client/verta/verta/_cli/registry/update.py

```diff
@@ -30,4 +32,34 @@ def update_model(model_name, label, workspace):
 def update_model_version(model_name, version_name, label, model, artifact, workspace):
     """Create a new registeredmodelversion entry.
     """
-    pass
+    client = Client()
+
+    try:
+        registered_model = client.get_registered_model(model_name, workspace=workspace)
+    except ValueError:
+        raise click.BadParameter("model {} not found".format(model_name))
+
+    try:
+        model_version = registered_model.get_version(name=version_name)
+    except ValueError:
+        raise click.BadParameter("version {} not found".format(version_name))
+
+    if model_version is None:
+        raise click.BadParameter("version {} not found".format(version_name))
+
+    if label is not None:
+        for l in label:
+            model_version.add_label(l)
+
+    if model is not None:
+        model_version.log_model(model, True)
+
+    if artifact is not None:
+        for (key, _) in artifact:
+            if key == "model":
+                raise click.BadParameter("the key \"model\" is reserved for model")
+
+        for (key, path) in artifact:
+            model_version.log_artifact(key, path, True)
```

Comment:
Could we check for collisions involving `model` and `artifact`'s keys  at the beginning of function, before any changes are made? That was if the user supplies a bad argument, they'll know before any difficult-to-reverse changes are made to their `ModelVersion`.

It's going to involve iterating through `model_version._msg.artifacts` (or you could add something like a `get_artifact_keys()`)

Reply:
Done. Also check if the user passes duplicate artifact keys. Add a test to reflect these.